docs/zh/latest/plugin-develop.md

```diff
@@ -378,7 +378,7 @@ end
 
 ### 注册控制接口
 
-如果你只想暴露 API 到 localhost 或内网，你可以通过 [Control API](../../en/latest/control-api.md) 来暴露它。
+如果你只想暴露 API 到 localhost 或内网，你可以通过 [Control API](../../zh/latest/control-api.md) 来暴露它。
```

Comment:
We don't need to go back to the grandparent directory and then go to his children. Just do what the English version does.